src/main/scala/edu/berkeley/ce/rockslicing/Block.scala

```diff
@@ -335,8 +394,10 @@ case class Block(center: (Double,Double,Double), faces: Seq[Face]) {
           Delaunay.Vector2(rotatedVertex(0), rotatedVertex(1))
         }.distinct.toList
 
-        // If z-component of normal is negative, order needs to be reversed to maintain clockwise orientation
-        if (face.c < -NumericUtils.EPSILON) {
+        // If normal is -z-axis, order needs to be reversed to maintain clockwise orientation since
+        // rotation matrix is identity matrix in this case - vectors are parallel. Rotation matrix
```

Comment:
Didn't you say this was when the vectors are pointing in opposite directions, or something like that?